content/productivity/crawl-planets/plugin.js

```diff
@@ -129,8 +171,9 @@ function capturePlanets(fromId, minCaptureLevel, maxDistributeEnergyPercent) {
   const candidates_ = df.getPlanetsInRange(fromId, maxDistributeEnergyPercent)
     .filter(p => (
       p.owner !== df.account &&
-      p.owner === "0x0000000000000000000000000000000000000000" &&
-      p.planetLevel >= minCaptureLevel
+      players.includes(p.owner) &&
+      p.planetLevel >= minCaptureLevel &&
+      p.planetType === planetType
```

Comment:
In future we can modify selection to list select few from list. Example: selection = add element like in repeat attack plugin